frontend/public/components/monitoring.tsx

```diff
@@ -251,18 +251,18 @@ const SeverityCounts: React.FC<{ alerts: Alert[] }> = ({ alerts }) => {
   );
 };
 
-const StateCounts: React.FC<{ alerts: PrometheusAlert[] }> = ({ alerts }) => {
-  const counts = _.countBy(alerts, 'state');
+export const StateCounts: React.FC<{ alerts: PrometheusAlert[] }> = ({ alerts }) => {
+  const counts = _.mapKeys(_.countBy(alerts, 'state'), (v, key) => _.toLower(key));
```

Comment:
Shouldn't need to convert to lower case (same as https://github.com/openshift/console/pull/5785/files#r452628730)